### DISCUSSION TITLE
When will feature/MinorMajorModLimits be merged to master and deployed?

### DISCUSSION THREAD

**Discussion by tonysathre** [Mijago/D2ArmorPicker#63] [General]
Just curious when this feature will be available.

Thanks

**Comment by Mijago** [comment]
Hi, I wanted to start developing it next week. The problem is that it is basically a complete rewrite of the generation logic, so it may take some time.

**Reply by tonysathre** [reply]
Looking forward to it!

Thanks

**Reply by Mijago** [reply]
I created this mockup yesterday:
![image](https://user-images.githubusercontent.com/3903469/147922185-073b247b-b5f0-4307-8bd7-e1ba3452f863.png)

My description for it (sorry for how it is written, I just wrote it fast in a chat and copied it here):
```
so the idea is as follows:
you have five equipment slots. Thus five rows.
For each row you
... can select the amount of stat points you want to use. This limits what kind of mod fits on the armor (if you set it to 3 it can not use major recov/int)
... can select the element. 
... can select a modslot/perk you want.

per default you just select what you want in general, ignoring the position of the slot. this means that you can select 3 GoS slots and they can occur on a class item, leg and helmet, ignoring position. BUT you can fix the location of the slot/element. For example I need an artificer slot on my gauntlets: I select artificer in the gauntlet row and press "lock for gauntlets"

No lock for the stat limit as it doesn't matter.
```

Of course I'd add a small description at the top/bottom, including a small help text.
Is this too complex?

**Reply by tonysathre** [reply]
Sorry for the late reply. I don't think this is too complex. I'd prefer it be more flexible like this so we can really tailor our builds.
